flow-base-api/src/main/java/com/aevi/sdk/flow/service/BaseApiService.java

```diff
@@ -48,6 +48,10 @@ protected void attachBaseContext(Context base) {
         internalData.setSenderPackageName(getPackageName());
     }
 
+    protected String getInternalData(@Nullable InternalData senderInternalData, String flowStage) {
```

Comment:
Can't see for sure - but is this called? It looks like the internal data is passed into the processRequest method?

Reply:
Yeap its used in `ActivityProxyService` and `BasePaymentFlowService` to get the stageName.

Might come in handy later if we need for anything else